agent/apps/ai_guardrails/deployment/urls_additions.py

```
@@ -0,0 +1,12 @@
+"""
+URL additions for ai_guardrails
+"""
+
+from django.urls import path, include
+
+from django.urls import path, include
```

Ruff (0.12.2)

Redefinition of unused `path` from line 5

Remove definition: `path`

(F811)